capi_problems.rst

```diff
@@ -7,7 +7,8 @@ An Evaluation of Python's C API
 Abstract
 ========
 
-This document describes our shared view of the C API, with an emphasis
+This document describes our shared view of the public C API
+intended to be consumed by external parties, with an emphasis
```

Comment:
I think this is ambiguous about what is intended to be consumed by external parties (the C API or this document).